tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -442,6 +448,25 @@ void CreateLineProbesFile(ExplorationTestDescription testDescription)
         }
     }
 
+    [CanBeNull]
+    static Assembly LoadAssemblySafe(string testAssemblyPath)
+    {
+        try
+        {
+            return Assembly.LoadFile(testAssemblyPath);
+        }
+        catch (BadImageFormatException e)
+        {
+            // ignore
+        }
+        catch (Exception e)
+        {
+            Logger.Warning($"Fail to load assembly: {testAssemblyPath}");
```

Comment:
nit: we could include the exception in the log, that'll come in handy if it happens